config/q-light/q-light_puck.xml

```diff
@@ -0,0 +1,150 @@
+	    <Item value="0" label="Device does not save the state before a power failure, it returns to off position"/>
+      <Item value="1" label="Device restores it's state before power failure"/>
+    </Value>
+    <Value genre="config" instance="1" index="8" value="0" label="Switch type" size="1" min="0" max="2" type="list">
+      <Help>
+Choose between momentary, ON/OFF and roller blind switch.
+
+Available settings:
+0 – momentary (Push) switch.
+1 – ON/OFF switch. 
+2– roller blind switch-two switchs operate the device(S1 to brighter, S2 to dim
+	  </Help>
+	  <Item value="0" label="momentary (Push) switch"/>
+    <Item value="1" label="ON/OFF switch"/>
+	  <Item value="2" label="roller blind switch-two switchs operate the device(S1 to brighter, S2 to dim)"/>
+    </Value>
+    <Value genre="config" instance="1" index="9" value="0" label="Switch functionality of S1 and S2" size="1" min="0" max="1" type="list">
+      <Help>Set the desired value from 0 to 1 to turn on/off the memory function.Setting this value to 0 turns off the dimmer's Memory fucntion.Setting this value to 1 truns on the dimmer's Memory function.</Help>
+	    <Item value="0" label="Standard mode"/>
+      <Item value="1" label="S1 operates as S2, S2 operates as S1"/>
+    </Value>
+    <Value genre="config" instance="1" index="10" value="1" label="S1 Scene ID set" units="step" size="1" min="1" max="255" type="byte">
+      <Help>
+When you press S1 five times,the light is dimmed to the brightness set in Scene ID 3 .
+
+S1 Scene ID is set by this configuration.
+
+Note: The actual action of each SCENE ID are set by SCE-NE_ACTUATOR_CONF(0x26) command.
+	  </Help>
+    </Value>
+    <Value genre="config" instance="1" index="11" value="2" label="S1 Scene ID set" units="step" size="1" min="1" max="255" type="byte">
+      <Help>
+When you press S1 five times,the light is dimmed to the brightness set in Scene ID 5.
+
+
+S2 Scene ID is set by this configuration.
+
+Note: The actual action of each SCENE ID are set by SCE-NE_ACTUATOR_CONF(0x26) command.
+	  </Help>
+    </Value>
+    <Value genre="config" instance="1" index="12" value="0" label="Timer functionality (auto - off)" units="seconds" size="2" min="0" max="32767" type="short">
+    <Help>This parameter allows to automatically switch off the device after specified time from switching on the light source. It may be useful when the Zerodim Z-Wave is installed in the stairway.
+
+Available settings: 
+0 - Function disabled. 
+1-32767 - time to turn off measured in seconds (1s-9.1h)
+    </Help>
+    </Value>
+  </CommandClass>
+
+  <CommandClass id="133">
+    <Associations num_groups="5">
+      <Group index="1" label="Lifeline" max_associations="1" />
+      <Group index="2" label="On/Off(S1) / Mirror of endpoint 1, group 2" max_associations="5" auto="true" />

```

Comment:
Are you sure you need each Association Group as Auto? Does the device not send everything to lifeline? (it should)